ui/app/AppLayouts/Profile/views/wallet/NetworksView.qml

```diff
@@ -1,48 +1,103 @@
 import QtQuick 2.13
 
 import shared.status 1.0
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import utils 1.0
 
 import "../../stores"
-import utils 1.0
+import "../../controls"
 
-Column {
+Item {
     id: root
-    anchors.top: parent.top
-    anchors.topMargin: 64
-    anchors.left: parent.left
-    anchors.right: parent.right
+    signal goBack
 
     property WalletStore walletStore
 
-    signal goBack
-
-    ListView {
-        id: layer1List
-        model: walletStore.layer1Networks
-        width: parent.width
-        height: layer1List.childrenRect.height
-        delegate: StatusSettingsLineButton {
-            text: model.chainName
+    StatusFlatButton {
+        id: backButton
+        anchors.top: parent.top
+        anchors.topMargin: 20
```

Comment:
this maybe instead be 16, 24, usually they are standard. but really dont know if this is the design leave it as it is

Reply:
done